staging/src/k8s.io/apiserver/pkg/authentication/request/bearertoken/bearertoken.go

```diff
@@ -45,7 +45,7 @@ func (a *Authenticator) AuthenticateRequest(req *http.Request) (*authenticator.R
 		return nil, false, nil
 	}
 	parts := strings.SplitN(auth, " ", 3)
-	if len(parts) < 2 || strings.ToLower(parts[0]) != "bearer" {
+	if len(parts) < 2 || !strings.EqualFold(parts[0], "bearer") {
```

Comment:
This seems like it could introduce a vulnerability, since we're now accepting unicode values that wouldn't have been accepted before and may not be expected by other tooling.

I'm not sure if there's anywhere in the code where we _want_ to be unicode aware for string comparison, maybe in some cases of display strings.